### DISCUSSION TITLE
banned for using this addon

### DISCUSSION THREAD

**Discussion by D0nHuan** [tarampampam/random-user-agent#373] [Q&A]
banned for using this addon, this is legit to ban me for use addon and switch user-agent?

**Comment by tarampampam** [comment]
Who and where banned you?

**Reply by D0nHuan** [reply]
grindinggear[.]com
_### Hi there,

Thanks for getting back to us.

Your account has been banned for malicious User Agent changing when sending requests to our websites.

If you can give us your word that this will stop, we can consider sending an appeal to have your account unbanned.

Kind regards,
Nav_

**Reply by tarampampam** [reply]
Some resources may do this to protect themselves from illegal usage, scraping, and so on. This is unique for each case.

**Reply by D0nHuan** [reply]
but im common internet user, just using this addon and do nothing more, they banned my account of legal right?

**Reply by tarampampam** [reply]
It looks like an edge case, you can decrease the auto-renewal interval, or disable this addon for this site

**Reply by D0nHuan** [reply]
sad news for me, coz i do nothing wrong, can i addressed this problem to google, coz i download this addon from official google store site?

**Reply by D0nHuan** [reply]
im using this addon many years, have google, youtube, facebook and etc accounts, they not banned me, but this little company banned for no reason (they banned in game client, but addon work only in my browser, this is terrible)

**Reply by tarampampam** [reply]
You can try, of course. But you should understand - this addon changes the default browser behavior, based on the addon settings. And it can have some Implications (such as https://github.com/tarampampam/random-user-agent/issues/304, for example)

**Reply by D0nHuan** [reply]
sorry but my level of programming languages like a dog seeing first time a vacuum cleaner

**Reply by tarampampam** [reply]
I'm sorry for your case :(
Could you try to message this game support? Maybe all not so bad as imagined?

**Reply by D0nHuan** [reply]
im try, but still banned, from last reply they wrote...
_Ban details
Reason: "Breaching Terms of Use"
Started: Oct 27, 2022_
anyway thanks for answer!

**Reply by tarampampam** [reply]
You're welcome! 🤗
